docs/conversion/sakai_21_mysql_conversion.sql

```diff
@@ -1,2 +1,7 @@
 -- clear unchanged bundle properties
 DELETE SAKAI_MESSAGE_BUNDLE from SAKAI_MESSAGE_BUNDLE where PROP_VALUE is NULL;
+
+-- SAK-43826 : Rubrics: Support weighted criterions
+
+ALTER TABLE RBC_RUBRIC ADD COLUMN WEIGHTED bit(1) NOT NULL DEFAULT 0;
```

Comment:
The case for this is wrong - right?
See https://github.com/sakaiproject/sakai-reference/pull/91/files
RBC_RUBRIC -> rbc_rubric
RBC_CRITERION -> rbc_criterion

Reply:
Sorry, didn't know that jira.
I updated the PR